hack/windows/Prepare-Node.ps1

```diff
@@ -50,6 +56,20 @@ If (Get-Service kubelet -ErrorAction SilentlyContinue) {
     exit 0
 }
 
+if ($ContainerRuntime -eq "Docker") {
+    if (-not(Test-Path "//./pipe/docker_engine")) {
+        Write-Error "Docker service was not detected - please install start Docker before calling
+PrepareNode.ps1 with -ContainerRuntime Docker"
+        exit 1
+    }
+} elseif ($ContainerRuntime -eq "ContainerD") {
+    if (-not(Test-Path "//./pipe/containerd-containerd")) {
+        Write-Error "ContainerD service was not detected - please install and start ContainerD
+before calling PrepareNode.ps1 with -ContainerRuntime ContainerD"
+        exit 1
+    }
```

Comment:
would it need another else if none of the value in ContainerRuntime provided!  else {
    Write-Error "Invalid value for -ContainerRuntime parameter. Allowed values are 'Docker' and 'ContainerD'."
    exit 1
}

Reply:
I don't think we need an error case ...There is an error generated automatically if user enters invalid value due to ValidaSet being used.

Error: Cannot validate argument on parameter 'ContainerRuntime'. The argument <invalid value> does not belong to the  set "ContainerD,Docker" specified by the ValidateSet attribute.